commerce-product-definitions-web/build.gradle

```diff
@@ -6,7 +6,7 @@ dependencies {
 	compileOnly group: "com.liferay", name: "com.liferay.expando.taglib", version: "2.0.0"
 	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib", version: "3.0.0"
 	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib.clay", version: "1.0.0"
-	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib.soy", version: "3.0.0"
```

Comment:
this is necessary to use the new attribute "useNAmespace" in the soy:render taglib

Reply:
@marco-leo only `2.2.0` should be necessary. `3.0.0` is a 7.2-version of `com.liferay.frontend.taglib.soy` -- when it was backported to `7.1`, it was released as version `2.2.0`, so that's why I've reverted to that version here.